src/payment/PaymentPage.jsx

```diff
@@ -40,29 +40,48 @@ class PaymentPage extends React.Component {
     );
   }
 
+  renderBasket() {
+    return (
+      <React.Fragment>
+        <ul>
+          <li>showVoucherForm: {JSON.stringify(this.props.showVoucherForm)}</li>
+          <li>paymentProviders: {JSON.stringify(this.props.paymentProviders)}</li>
+          <li>orderTotal: {this.props.orderTotal}</li>
+          <li>lineDiscount: {this.props.lineDiscount}</li>
+          <li>sdnCheck: {JSON.stringify(this.props.sdnCheck)}</li>
+          <li>lineTotal: {this.props.lineTotal}</li>
+          <li>products: {JSON.stringify(this.props.products)}</li>
+          <li>voucher: {JSON.stringify(this.props.voucher)}</li>
+        </ul>
+      </React.Fragment>
+    );
+  }
+
   render() {
     const {
       loading,
       loadingError,
-      payment,
+      orderTotal,
     } = this.props;
     const loaded = !loading && !loadingError;
-    const hasPayments = payment.total;
+    const basketHasItems = orderTotal !== undefined;
```

Comment:
I wonder if the API response should be more explicit instead of putting this business logic here, i.e. return a `basketHasItems` value in the response.

Reply:
We could also use the length of `products`

Reply:
Exactly my point. That's a decision better made by the backend.

Reply:
Something to consider down the road...